kong/clustering/services/sync/rpc.lua

```diff
@@ -79,13 +80,14 @@ function _M:init_cp(manager)
 
     -- { default = { version = 1000, }, }
     local default_namespace_version = default_namespace.version
+    local node_info = assert(kong.rpc:get_peer_info(node_id))
```

Comment:
if assertion failed, it will corrupted one single timer runtime, so it's safe for CP.

*  kong/clustering/rpc//socket.lua: callback is runned in a timer
```
        -- call dispatch
        local res, err = kong.timer:named_at(string_format("JSON-RPC callback for node_id: %s, id: %d, method: %s",
                                                           self.node_id, payload.id, payload.method),
                                                           0, _M._dispatch, self, dispatch_cb, payload)

```